docs/build.js

```diff
@@ -73,13 +78,13 @@ async function run_with_theme(page, is_dark = false) {
             )
         );
         console.log(JSON.stringify(new_config));
+
         await page.evaluate(async (config) => {
             const viewer = document.querySelector("perspective-viewer");
             await viewer.reset();
             await viewer.restore(config);
         }, new_config);
 
-        await page.waitForSelector("perspective-viewer:not([updating])");
```

Comment:
What is the story behind this + [lines 52-56 in this file](https://github.com/finos/perspective/pull/2502/files#diff-4e0e40e9d4c5f482f9ba06ed27a0747b5ddf5fec750a3a34709fca59a7f2f3e0R52)?